libs/sdk-ui/src/base/localization/bundles/en-US.json

```diff
@@ -177,6 +177,21 @@
         "comment": "Means comparison of two values - for example '100 (this week) VERSUS 10 (last week)'.",
         "limit": 0
     },
+    "visualizations.headline.comparison.title.change": {
+        "value": "Change",
+        "comment": "The default label value of comparison when calculating as change.",
```

Comment:
I would change this to "The label of dropdown option that specifies arithmetic operation to be percentual change between two values."

Reply:
Updated!